packages/ses/src/whitelist.js

```diff
@@ -940,22 +944,28 @@ export const whitelist = {
       // See https://github.com/tc39/proposal-array-find-from-last
       findLast: 'boolean',
       findLastIndex: 'boolean',
+      // https://github.com/tc39/proposal-change-array-by-copy
       toReversed: 'boolean',
       toSorted: 'boolean',
       toSpliced: 'boolean',
       with: 'boolean',
+      // https://github.com/tc39/proposal-array-grouping
       group: 'boolean',
       groupToMap: 'boolean',
+      groupBy: 'boolean',
     },
     // See https://github.com/tc39/proposal-array-find-from-last
     findLast: fn,
     findLastIndex: fn,
+    // https://github.com/tc39/proposal-change-array-by-copy
     toReversed: fn,
     toSorted: fn,
     toSpliced: fn,
     with: fn,
-    group: fn,
-    groupToMap: fn,
+    // https://github.com/tc39/proposal-array-grouping
+    group: fn, // Not in proposal? Where?
+    groupToMap: fn, // Not in proposal? Where?
+    groupBy: fn,
```

Comment:
I believe these have been withdrawn. Vendors unshipped `group` at least on account of web compat concerns. Judging by reactions in TC39’s Matrix chat, Mozilla and Google have both lost their appetite for any further properties on `Array.prototype`.